zrml/prediction-markets/src/mock.rs

```diff
@@ -44,6 +44,8 @@ use sp_runtime::{
     DispatchError, DispatchResult,
 };
 use std::cell::RefCell;
+#[cfg(feature = "parachain")]
+use zeitgeist_primitives::types::XcmAsset;
```

Comment:
We already have that feature above and could combine all conditional imports.

Reply:
You mean with `cfg_if`?

Reply:
Like https://github.com/zeitgeistpm/zeitgeist/blob/2d0277bea7ba5796733b1db4db484dc73f30e23c/zrml/prediction-markets/src/lib.rs#L58-L62

Reply:
9c7d94edeca54627f07b3a77f52e8ca82752ac25